src/QueryBuilderRequest.php

```diff
@@ -38,8 +38,8 @@ public function includes(): Collection
 
         $includeParts = $this->getRequestData($includeParameterName);
 
-        if (! is_array($includeParts)) {
-            $includeParts = explode(static::getIncludesArrayValueDelimiter(), $this->getRequestData($includeParameterName));
+        if (! is_array($includeParts) && ! is_null($includeParts)) {
```

Comment:
Maybe you could change this to `is_string($includeParts)` and eliminate the other checks. I think this would avoid the warning as well as improve readability.

Reply:
Agreed. Just saw this is the conditional on line 82. 👍